src/registry/licenseplate/preview.tsx

```diff
@@ -0,0 +1,29 @@
+import {LicensePlateComponentSchema} from '@open-formulieren/types';
+
+import {TextField} from '@/components/formio';
+
+import {ComponentPreviewProps} from '../types';
+
+/**
+ * Show a formio iban component preview.
+ *
+ * NOTE: for the time being, this is rendered in the default Formio bootstrap style,
+ * however at some point this should use the components of
+ * @open-formulieren/formio-renderer instead for a more accurate preview.
+ */
+const Preview: React.FC<ComponentPreviewProps<LicensePlateComponentSchema>> = ({component}) => {
+  const {key, label, description, tooltip, validate = {}, multiple} = component;
+  const {required = false} = validate;
```

Comment:
isn't it because of the `validate = {}` default? because `component.validate` itself can be undefined.

Reply:
For some reason, using `{}` as a default value doesn't seem to play well if `validate` is overridden in the types library (either as `validate` or `validate?`):
```ts
const {key, label, description, tooltip, validate = {}, multiple} = component;  // `validate` is of type `{}`
const {key, label, description, tooltip, validate, multiple} = component;  // `validate` is of type OFValidateOptions<Validator | "plugins"> & { "^[a-zA-Z0-9]{1,3}\\-[a-zA-Z0-9]{1,3}\\-[a-zA-Z0-9]{1,3}$";}
```

Fortunately types define `validate` as being required, so we can safely remove the default value.

May be related: https://github.com/microsoft/TypeScript/issues/41189

Reply:
Yeah, perhaps this should be `= ({} satisfies OFValidate<...>)` in this case, right now it's probably interpreted as a literal.

Doing the destructuring differently here is acceptable for me :+1: